ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/SqlFormatterQueryRewrite.java

```diff
@@ -119,36 +113,8 @@ protected Void visitExpression(Expression node, Integer indent) {
 
     @Override
     protected Void visitQuery(Query node, Integer indent) {
-      if (node.getWith().isPresent()) {
```

Comment:
This seems exactly the same as the `SQLFormatter` method. How are these classes different?

Reply:
They were duplicate. removed this one.